File: Clase6/SoftProgSolution/SoftProgModel/src/main/java/pe/edu/pucp/softprog/rrhh/model/Area.java
package pe.edu.pucp.softprog.rrhh.model;
public class Area {
    private int idArea;
    private String nombre;
    private boolean activa;

    public int getIdArea() {
        return idArea;
    }

    public void setIdArea(int idArea) {
        this.idArea = idArea;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public boolean isActiva() {
        return activa;
    }

    public void setActiva(boolean activa) {
        this.activa = activa;
    }

    @Override
    public String toString() {
       return "AREA: " + idArea + ". " + nombre;
    }
}

File: Clase12/ProyectosNET/SoftProgSolution/SoftProgModel/RRHH/Area.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace SoftProgModel.RRHH
{
    public class Area
    {
        private int _idArea;
        private String _nombre;
        private bool _activa;
        public Area() { }
        public Area(String nombre)
        {
            _nombre = nombre;
        }
        public int IdArea { get => _idArea; set => _idArea = value; }
        public string Nombre { get => _nombre; set => _nombre = value; }
        public bool Activa { get => _activa; set => _activa = value; }

        public override string ToString()
        {
            return _idArea + ". " + _nombre;
        }

    }
}
